src/backend/optimizer/plan/planner.c

```diff
@@ -85,11 +85,22 @@
 #include "optimizer/orca.h"
 #include "storage/lmgr.h"
 #include "utils/guc.h"
+#include "utils/builtins.h"
 
 #ifdef USE_ORCA
 extern void InitGPOPT();
 #endif
 
+#include "access/genam.h"
+#include "access/table.h"
+#include "access/htup_details.h"
+#include "catalog/catalog.h"
+#include "catalog/pg_rewrite.h"
+#include "catalog/pg_class_d.h"
+#include "parser/parsetree.h"
+#include "utils/rel.h"
+#include "tcop/tcopprot.h"
```

Comment:
put includes on top of `InitGPOPT `.

Reply:
why not in alphabet order ? for example: `#include "utils/builtins.h"`

Reply:
Fixed.